api/org/commcare/api/persistence/SqlSandboxUtils.java

```diff
@@ -13,13 +13,28 @@
  * @author wspride
  */
 public class SqlSandboxUtils {
+
+    public static UserSqlSandbox getStaticStorage(String username, String path) {
+        System.out.println("Get STatic storage path: " + path);
+        return new UserSqlSandbox(username, path);
+    }
+
     public static UserSqlSandbox getStaticStorage(String username) {
-        return new UserSqlSandbox(username);
+        return new UserSqlSandbox(username, null);
     }
 
     public static void deleteDatabaseFolder(){
-        File databaseFolder = new File(SqliteIndexedStorageUtility.DATABASE_FOLDER);
-        if(databaseFolder.exists()){
+        deleteDatabaseFolder(null);
```

Comment:
It makes way more sense to me to set the argument here to be SqliteIndexedStorageUtility.DATABASE_FOLDER rather than swithing on Null below.